src/pages/UserRenewal/UserRenewal.tsx

```diff
@@ -141,8 +142,19 @@ export default function () {
       })
       .catch((error) => {
         notificationLogic(
-          "info",
-          "Сталася помилка при зверненні до адміністрації міста"
+          "error",
+          "Сталася помилка при зверненні до адміністрації міста, спробуйте пізніше"
```

Comment:
Maybe it's worth to move that string to some const variable and just reuse it?

Reply:
Thx, added new notification string with new commit.